cypress/integration/recieve-notice/recieve-notice-checkin.spec.js

```diff
@@ -0,0 +1,185 @@
+        cy.getInstanceTypes({ limit: 1 }).then((res) => { testData.instanceType = res[0].id; });
+        cy.getLoanTypes({ limit: 1 }).then((res) => { testData.loanType = res[0].id; });
+      })
+
+      .then(() => {
+        cy.createInstance({
+          instance: {
+            instanceTypeId: testData.instanceType,
+            title: testData.instanceTitle,
+          },
+          holdings: [{
+            holdingsTypeId: testData.holdingType,
+            permanentLocationId: testData.location,
+            sourceId: testData.holdingSource,
+          }],
+          items: [
+            [{
+              barcode: ITEM_BARCODE,
+              missingPieces: '3',
+              numberOfMissingPieces: '3',
+              status: { name: 'Available' },
+              permanentLoanType: { id: testData.loanType },
+              materialType: { id: testData.materialType },
+            }],
+          ],
+        });
+      });
+
+    cy.getCirculationRules().then((res) => {
+      testData.baseRules = res.rulesAsText;
+      testData.ruleProps = CirculationRules.getRuleProps(res.rulesAsText);
+    });
+
+    loanPolicy.getApi({ limit: 1 }).then((loanPolicyRes) => {
+      testData.ruleProps.l = loanPolicyRes.body.loanPolicies[0].id;
+      testData.loanNoticeName = loanPolicyRes.body.loanPolicies[0].name;
+    });
+
+    cy.loginAsAdmin({ path: settingsMenu.circulationPatronNoticePoliciesPath, waiter: NewNoticePolicyTemplate.waitLoading });
+  });
+
+  afterEach('Deleting created entities', () => {
+    CheckInActions.createItemCheckinApi({
+      itemBarcode: ITEM_BARCODE,
+      servicePointId: testData.userServicePoint,
+      checkInDate: moment.utc().format(),
+    }).then(() => {
+      Users.deleteViaApi(userData.id);
+      PatronGroups.deleteViaApi(patronGroup.id);
+    });
+
+    cy.getInstance({ limit: 1, expandAll: true, query: `"items.barcode"=="${ITEM_BARCODE}"` })
+      .then((instance) => {
+        cy.deleteItem(instance.items[0].id);
+        cy.deleteHoldingRecord(instance.holdings[0].id);
+        cy.deleteInstanceApi(instance.id);
+      });
+
+    CirculationRules.deleteRuleApi(testData.baseRules);
+    NoticePolicyApi.deleteApi(testData.ruleProps.n);
+    NoticePolicyTemplateApi.getViaApi({ query: `name=${noticePolicyTemplate.name}` }).then((templateId) => {
+      NoticePolicyTemplateApi.deleteViaApi(templateId);
+    });
+  });
+
+  it('C347623 Check that user can receive notice with multiple items after finishing the session "Check in" by clicking the End Session button', { tags: [testTypes.smoke, devTeams.vega] }, () => {
+    NewNoticePolicyTemplate.startAdding();
+    NewNoticePolicyTemplate.checInitialState();
+    NewNoticePolicyTemplate.create(noticePolicyTemplate);
+    NewNoticePolicyTemplate.addToken(noticePolicyTemplate);
+    noticePolicyTemplate.body += '{{item.title}}';
+    NewNoticePolicyTemplate.save();
+    NewNoticePolicyTemplate.checkAfterSaving(noticePolicyTemplate);
```

Comment:
let's expand object properties into methods parameters to clear scope of testing. Or add check that that object contains expected properties...